stdlib/public/core/UnboundCapacity.swift

```diff
@@ -110,7 +121,9 @@ extension UnboundCapacity {
 
   /// Returns the small (`Base`) representation iff we have it and it has at
   /// least the given capacity.
-  internal func _small(minCapacity: Int) -> Base? {
+  ///
+  /// Michael: I made this public to squash linker errors in prototypes
```

Comment:
This one may of been over eager. I think the errors were because we disable access control to use internal pieces.